setup.py

```diff
@@ -13,7 +13,10 @@ def read(name):
         return f.read()
 
 # Used for the tar.gz/snap releases
-VERSION = "1.9.8"
+latest_release_url = f"https://api.github.com/repos/AdnanHodzic/auto-cpufreq/releases/latest"
+latest_release = requests.get(latest_release_url).json()
+VERSION =  latest_release["tag_name"]
```

Comment:
What's the benefit of doing this instead of explicitly referring to i.e: `VERSION = "1.9.8"`

As I don't think it's a bad idea to have a version defined in a single place.

Reply:
no its not a bad idea, i just tried to make it change automatically assuming the user installing the clonned version anyway manually or using --update.

Reply:
Yea, I understand, but let's put it back to `VERSION = "1.9.8"`. As sometimes I'll build snap packages with `*-beta` which I manually add during testing.

Reply:
okay got ti